client/signup/steps/plans/index.jsx

```diff
@@ -125,6 +144,30 @@ module.exports = React.createClass( {
 		);
 	},
 
+	plansFeaturesSelection: function() {
+		const {
+			flowName,
+			stepName,
+			positionInFlow,
+			signupProgressStore
+		} = this.props;
+
+		const translate = this.translate;
+
+		const headerText = translate( "Pick a plan that's right for you." );
```

Comment:
Oh just saw @mtias earlier note. So ignore my note above, and it looks like eslint is properly configured for this too:
```
		'quotes': [ 1, 'single', 'avoid-escape' ],
```